WebApplication/ClientApp/src/components/FetchData.js

```diff
@@ -51,9 +52,10 @@ export class FetchData extends Component {
     );
   }
 
-  async populateWeatherData() {
-    const response = await fetch('weatherforecast');
-    const data = await response.json();
-    this.setState({ forecasts: data, loading: false });
-  }
+    async populateWeatherData() {
+        const apiUrl = App.apiUrl;
+        const response = await fetch('weatherforecast');
+        const data = await response.json();
+        this.setState({ forecasts: data, loading: false });
+    }
```

Comment:
What is the point of this?

Reply:
It's a default VStudio React method which will be removed later on. (For now, it is used as an example)